dlc-manager/src/manager.rs

```diff
@@ -2267,4 +2293,29 @@ mod test {
             .on_dlc_message(&offer_message, pubkey())
             .expect_err("To reject the second offer message");
     }
+
+    #[test]
```

Comment:
that's not a great test, I think it'd be better to add some checks in the integrations tests.